src/amo/components/AddonVersionCard/index.js

```diff
@@ -97,22 +97,18 @@ export const AddonVersionCardBase = (props: InternalProps) => {
           ? { to: `/addon/${addon.slug}/license/` }
           : { href: license.url, prependClientApp: false, prependLang: false };
 
-        const licenseText = license.name
-          ? i18n.sprintf(
-              i18n.gettext(
-                'Source code released under %(linkStart)s%(licenseName)s%(linkEnd)s',
-              ),
-              {
-                licenseName: license.name,
-                // Keep the link placeholders so that we can use them to inject a
-                // `<Link />` using `replaceStringsWithJSX`.
-                linkStart: '%(linkStart)s',
-                linkEnd: '%(linkEnd)s',
-              },
-            )
-          : i18n.gettext(
-              'Source code released under %(linkStart)san unidentified license%(linkEnd)s',
-            );
+        const licenseText = i18n.sprintf(
+          i18n.gettext(
+            'Source code released under %(linkStart)s%(licenseName)s%(linkEnd)s',
+          ),
+          {
+            licenseName: license.name || 'Custom License',
```

Comment:
We should keep the previous condition I believe and have `Custom License` put inside a `gettext` call, otherwise we won't be able to localize this string.

Reply:
Updated!